.github/workflows/pytest.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Test with pytest
       run: |
-        pytest --cov=lineage --cov-report=xml
+        pytest --cov=lineage --cov-report=xml --cov-config=.coveragerc
```

Comment:
With the way you have it, this should be `--cov-config=.github/workflows/coveragerc`